src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,11 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <>
+    {comments.map(comment => (
+      <CommentInfo key={comment.id} comment={comment} />
+    ))}
+  </>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings